.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@
 :cl:
 add: Added new mechanics or gameplay changes
 add: Added more things
-expansion: Expands content of an existing thing
+expansion: Expands content of an existing feature
```

Comment:
```suggestion
expansion: Expands content of an existing system or feature
```

Reply:
Will this change break everything again?